src/mixins/date-mixin.ts

```diff
@@ -65,18 +65,19 @@ export default class DateMixin extends Vue {
   }
 
   /**
-   * Converts a Date object to a date string (YYYY-MM-DD) in Pacific timezone.
+   * Converts a Date object to a date string (YYYY-MM-DD)
+   * @param convertTimezone whether to convert the timzone to Pacific
    * @example "2021-01-01 07:00:00 GMT" -> "2020-12-31"
    * @example "2021-01-01 08:00:00 GMT" -> "2021-01-01"
    */
-  dateToYyyyMmDd (date: Date): IsoDatePacific {
```

Comment:
Since Vuetify 3 Datepicker returns a `Date`, it also includes timezone information and defaults the time to `00:00:00`. Our previous datepicker would ignore this since it would return a string. 

I tested the Vuetify 3 DatePicker for non-pacific timezones and this resulted in bizarre behaviour. Notice below that Dec 4 is selected in the picker, but the text field shows Dec 3. The console.log shows that the time is set to 12AM CST, which means when it converts to a pacific date in this mixin function, it sets the date back a day.

<img width="1238" alt="Screenshot 2023-12-06 at 8 51 08 AM" src="https://github.com/bcgov/bcrs-shared-components/assets/122323255/c30524ad-7092-4070-8ec2-11979d470570">

My solution is to update this function to allow the option of ignoring the timezone.

Reply:
Two things:
1. I'd rather not change the date mixin.
2. When this component is used in an app, it's using the app's date mixin (see comment above).

If this method isn't working for you then it means the `date` parameter may be invalid. Normally, a date object is UTC/GMT internally and has a tz for display purposes. Try some of these on `date` and see how it's set internally.

![image](https://github.com/bcgov/bcrs-shared-components/assets/10002889/47ed6f2b-e878-4a39-8be1-de9fc42b45b2)

As for this method, it should always convert a date into America/Vancouver time zone.